<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QWidget" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">
QLineEdit {
    border-width: 1px;
    border-style: solid;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
}
QComboBox {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
}
QFrame{
};
background-color:rgb(255, 255, 255);
border:1px solid rgba(227, 135, 193, 255);
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
border: 1px solid rgba(255 , 255 , 255, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>371</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background:none;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="pwdEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(255, 192, 207);
}</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loginButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="registButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
color:rgb(255,255,255);
border-radius: 5px;
border:none;
}QPushButton::hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(227, 135, 193, 255), stop:0 rgba(251, 171, 182, 255));
}
QPushButton::pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(227, 135, 193, 255), stop:1 rgba(251, 171, 182, 255));
}</string>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:none;
bottom:10px;
border: none;</string>
        </property>
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:none;
bottom:10px;
border: none;</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="jidEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:3px;
    border-bottom-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(255, 192, 207);
}</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
